corehq/apps/builds/models.py

```diff
@@ -152,6 +152,13 @@ def get_label(self):
         else:
             return None
 
+    def get_menu_item_label(self):
+        build_config = CommCareBuildConfig.fetch()
+        for item in build_config.menu:
+            if item.build.version == self.version:
```

Comment:
I'm comparing versions instead of `to_string()` because I found that `item.build.to_string()` would return "2.25.0/latest" but `self.to_string()` would return "2.25.0/\<build_number>". i.e. Despite the fact that the build was built against the same version/menu item, item.build != self, and self.latest != True.